npm/config/config.go

```diff
@@ -16,8 +16,10 @@ const (
 
 // DefaultConfig is the guaranteed configuration NPM can run in out of the box
 var DefaultConfig = Config{
-	WindowsNetworkName:    util.AzureNetworkName,
-	ResyncPeriodInMinutes: defaultResyncPeriod,
+	WindowsNetworkName:                   util.AzureNetworkName,
+	ApplyDataPlaneMaxBatches:             0,
```

Comment:
make sure these default are real numbers, if config map is not applied properly we should not be erroring out.